moveit_ros/planning/planning_request_adapter_plugins/src/add_time_optimal_parameterization.cpp

```diff
@@ -47,17 +47,56 @@ using namespace trajectory_processing;
 class AddTimeOptimalParameterization : public planning_request_adapter::PlanningRequestAdapter
 {
 public:
+  double path_tolerance = 0.1;
+  double resample_dt = 0.1;
+  double min_angle_change = 0.001;
+
   AddTimeOptimalParameterization() : planning_request_adapter::PlanningRequestAdapter()
   {
   }
 
-  void initialize(const ros::NodeHandle& /*nh*/) override
+  void initialize(const ros::NodeHandle& nh) override
   {
+
+    
+    double path_tolerance_override;
+    if (!nh.getParam("/path_tolerance", path_tolerance_override))
```

Comment:
please move the param names to string constants (if you need to reuse them it's easier to avoid typos) and also note that other plugins omit the "/" in front. This reads the parameters relative to some position internal to the `NodeHandle` and allows to have multiple planning pipelines with different parameters

see here for an example:
 https://github.com/ros-planning/moveit/blob/253a2177f23c2d5e2e3fa415c75f720c9d6a98a5/moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_collision.cpp#L61

Reply:
Great. I'll give that a shot.